opennft/opennft.py

```diff
@@ -468,6 +505,33 @@ def onChangeDataType(self):
         self.cbgetMAT.setChecked(self.cbgetMAT.isChecked() and self.cbDataType.currentText() == 'DICOM')
         self.cbgetMAT.setEnabled(self.cbDataType.currentText() == 'DICOM')
 
+    # --------------------------------------------------------------------------
+    def showPluginDlg(self):
```

Comment:
Greetings! This is my first post here, so I'll try to be specific in it.

I discovered, that every time user opens Plugin window - new instance of PluginWindow is created. So it confused me, when I chose plugins, click OK button, then Setup and after that - open Plugins again. There were no plugins chosen. As a consequence of this I've got an error.

I suggest to make Plugin window the same way as rtQA window - created once for whole session and user interact with before setup.

Reply:
Sounds reasonable. I will make the changes

Reply:
A question:
rtQA window is (re)created for each setup. What do you think would be the most sensible approach?
- (re)create clear Plugin window at each Setup
- create Plugin window only once at app start and always keep plugin statuses unless user changes them

Reply:
I think the second option is better for Plugin window. Moreover, it might be a part of initialization block

Reply:
Done it.